ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py

```diff
@@ -2037,14 +2038,34 @@ def test_paddle_amax(
     frontend,
     test_flags,
 ):
+  
+
+=======
```

Comment:
Hello again! You will not be able to merge the PR yourself. Only reviewers can do that. Feel free to ask whatever you need, we are here to help!

Also, it seems like there are some residual lines from when you merged the new changes. Maybe this is what the lint test was flagging. I recommend that you copy and save your changes in a text editor so you don't lose them, and update your branch to restore the file. Then, you can add your changes again if you need to, or simply push everything again

Reply:
I think I resolved the changes by making a new set of commits